modules/t/test_division/homo_sapiens.json

```diff
@@ -0,0 +1,20 @@
+{
+    "homo_sapiens" : {
+        "core" : {
+            "regions" : [
+                ["1",  27830782, 28871267],
+                ["15", 82988441, 84039842],
+                ["17",   137526,  1400222],
+                ["21", 17513043, 18485879],
+                ["Y",   5000226,  6249019],
+                ["LRG_1143", 82534750, 82545544, "lrg"],
+                ["LRG_885",   1737836,  1757265, "lrg"],
+                ["LRG_18",   44280880, 44300227, "lrg"]
+            ],
+            "adaptors" : [
+                {"name" : "gene", "method" : "fetch_all_by_Slice", "args" : []},
```

Comment:
I have found in `clone_core_database.pl` that `fetch_all_by_Slice` is the default "method", and that "args" defaults to an empty list. I think you can skip both.